unfetter-discover/unfetter-ui
# Issue 884 (incomplete - ignore for now)

Fixes unfetter-discover/unfetter:884

Introduction of summary page (more of a UI component playground) for Assessments 3.0.  There is a bunch of code still in place taken from the current assessments, and this will be removed or refactored over time as we continue implementing UI components and workflows for A3.0.

# To test:

1. Pull this PR
2. Restart stack (or just UI with `docker restart unfetter-ui`)
3. Click navigator control in header and select "Assessments 3.0"
4. Verify basic summary display of first object assessment in the mongo DB (with name of "Sysmon Assessment"
    a. If there are no object assessments in the DB, the first page of the create wizard for Assessments will display
